pipenv/core.py

```diff
@@ -903,6 +903,7 @@ def do_create_virtualenv(project, python=None, site_packages=None, pypi_mirror=N
         Path(sys.executable).absolute().as_posix(),
         "-m",
         "virtualenv",
+        "--creator=venv",
```

Comment:
This change did solve the issue for my ubuntu 22.04 -- I am not sure if that `-m venv` is equivalent to this, do you know?

Reply:
The built in module `venv` is a trimmed down version of `virtualenv`: https://virtualenv.pypa.io/en/latest/.
We can't use this approach immediately anyway because it only supports one python version.

To use this approach, we will have to detect the python version by ourselves. That's because you can only do:

```
python3.X -m venv ...
```
Instead of:
 ```
python -m virtualenv --python ....
```